pkg/networkpolicy/metrics.go

```diff
@@ -33,7 +33,7 @@ var (
 	packetCounterVec = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Name: "packet_count",
 		Help: "Number of packets",
-	}, []string{"protocol", "family"})
+	}, []string{"protocol", "family", "verdict"})
```

Comment:
If we tossed the tossed the whole packet 5 part tuple in here,  we  could use a cli tool to generate network policies for the last amount of time all your daemonsets were up. But that seem like maybe an optional metric to leave on so leaving for another PR.

Reply:
Yep exactly and it doesn't even catch the labels so would almost certainly have to come through logs to build something like this. 

Verdict is still helpful though as people can alert on continuos blocking as its either a misconfiguration or an attack.

Reply:
agree